inc/operatingsystemkernelversion.class.php

```diff
@@ -62,10 +62,10 @@ function displaySpecificTypeField($ID, $field = []) {
       }
    }
 
-   function getRawName() {
+   function computeFriendlyName() {
       $kernel = new OperatingSystemKernel();
-      $kname = $kernel->getRawName();
-      $kvname = parent::getRawName();
+      $kname = $kernel->getFriendlyName();
+      $kvname = parent::getFriendlyName();
```

Comment:
Will loop unless you call something like:
```suggestion
      $kvname = parent::computeFriendlyName();
```